Update results to weigh by citation counts (#123)
greenelab/iscb-diversity-manuscript: content/30.methods.md

To automate and generalize these tasks, we created the [pubmedpy](https://github.com/dhimmel/pubmedpy) Python package.

From PubMed, we compiled a catalog of 176,773 journal articles that were published from 1993 through 2019, written in English, and tagged with the MeSH term ["computational biology"](https://pubmed.ncbi.nlm.nih.gov/?term=%22Computational+Biology%22%5BMeSH+Terms%5D+), which is [equivalent](https://www.ncbi.nlm.nih.gov/mesh/68019295) to "bioinformatics" and includes categories such as genomics and systems biology (via PubMed's term explosion to include subterms).
Excluding 663 articles with no author information and years with less than 200 articles/year, we analyzed 176,110 articles from 1998--2019.

We extracted the number of times an article has been cited by PubMed Central articles from the `PmcRefCount` of the PubMed DocSum XML records.
Under the rationale that authors of more impactful articles have higher chance of being recognized, we used the PubMed Central citation count as a relative estimate of an article's impact to weigh each author's prediction probability (please see the Statistical Analysis subsection).

> However, we note the potential bias that comes with this citation count weighing approach,

Reviewer: ```suggestion
However, we note a potential bias for citation count weighing:
```

> such as that female scientists of different disciplines are cited less often [@arxiv:2001.01002, @doi:10.1002/ece3.4993, @doi:10.1038/s41550-017-0141].

Reviewer: ```suggestion
female scientists of different disciplines are cited less often than their male counterparts [@arxiv:2001.01002, @doi:10.1002/ece3.4993, @doi:10.1038/s41550-017-0141].
To the extent that there may be a bias in citation practices, we should expect that the bias for honorees would align with one for citations, meaning that our analysis would understate the severity of disparities.
```

From PubMed, we compiled a catalog of 176,773 journal articles that were published from 1993 through 2019, written in English, and tagged with the MeSH term ["computational biology"](https://pubmed.ncbi.nlm.nih.gov/?term=%22Computational+Biology%22%5BMeSH+Terms%5D+), which is [equivalent](https://www.ncbi.nlm.nih.gov/mesh/68019295) to "bioinformatics" and includes categories such as genomics and systems biology (via PubMed's term explosion to include subterms).
Excluding 663 articles with no author information and years with less than 200 articles/year, we analyzed 176,110 articles from 1998--2019.

We extracted the number of times an article has been cited by PubMed Central articles from the `PmcRefCount` of the PubMed DocSum XML records.
Under the rationale that authors of more impactful articles have higher chance of being recognized, we used the PubMed Central citation count as a relative estimate of an article's impact to weigh each author's prediction probability (please see the Statistical Analysis subsection).
However, we note the potential bias that comes with this citation count weighing approach,

> such as that female scientists of different disciplines are cited less often [@arxiv:2001.01002, @doi:10.1002/ece3.4993, @doi:10.1038/s41550-017-0141].

Reviewer 2: ```suggestion
such as that female scientists of different disciplines are cited less often [@arxiv:2001.01002; @doi:10.1002/ece3.4993; @doi:10.1038/s41550-017-0141].
```
